### ISSUE TITLE
Token streaming doesn't work

### ISSUE THREAD

**Issue by LoganDark** [henk717/KoboldAI-Client#517]
[kobold_debug.json](https://github.com/henk717/KoboldAI/files/15272513/kobold_debug.json)

For some reason token streaming just does not work. It's enabled and the actual terminal output from the server updates every token but no messages are actually sent over websocket to the UI so it can't be displayed until the response is complete. No idea what is going on.

I'm on the latest United commit 1e985ed51bcdde506c223a93017aa05647792063.

**Comment by henk717** [comment]
What kind of model / api was it hooked up to?

**Comment by LoganDark** [comment]
Thought that would be in the debug json, but I've tried with both LLaMA 2 and Mixtral 8x7B in GGUF format, running on KoboldCPP (with cuBLAS and full offload to a 3090). I'm using the KoboldAI United UI (localhost:5000, not lite).

**Comment by henk717** [comment]
United can't stream over the API thats why streaming is missing.

**Comment by LoganDark** [comment]
What do you mean it can't stream over the API? So it can't stream at all?

**Comment by henk717** [comment]
It can stream when you use huggingface based models in the main UI.

**Comment by LoganDark** [comment]
So I can't use my 3090 to run models? Or I can't use GGUF files?

**Comment by henk717** [comment]
You can't use GGUF's combined with United combined with streaming.
You can use it when you directly use Koboldcpp in its own bundled KoboldAI Lite.

**Comment by LoganDark** [comment]
OK, so the solution is to not use GGUF then? the lite UI is mostly unusable for me (it works fine, it just has an awful user experience)

**Comment by henk717** [comment]
Yes, the backends built in to KoboldAI United should work (Huggingface, exllama2)
